config/initializers/assets.rb

```diff
@@ -3,15 +3,16 @@
 # Version of your assets, change this if you want to expire all your assets.
 Rails.application.config.assets.version = "1.0"
 
-# We do not add to asset paths because or a webpacker compatibility
-# https://github.com/rails/webpacker/issues/1285#issuecomment-539553734
 # Add additional assets to the asset load path.
 # Rails.application.config.assets.paths << Emoji.images_path
-# Add Yarn node_modules folder to the asset load path.
-# Rails.application.config.assets.paths << Rails.root.join('node_modules')
 
 # Precompile additional assets.
 # application.js, application.css, and all non-JS/CSS in the app/assets
 # folder are already added.
 
 # Rails.application.config.assets.precompile += %w( admin.js admin.css )
+
+Rails.application.config.assets.paths << Rails.root.join("app/assets/builds")
+Rails.application.config.assets.paths << Rails.root.join('assets')
+Rails.application.config.assets.paths << Rails.root.join("lib", "liquid", "template", "buyer_side")
+Rails.application.config.assets.paths << Rails.root.join("vendor", "assets", "images")
```

Comment:
Copied directly from here: https://github.com/3scale/porta/blob/00e4733096ebb2dc248acc388dda984dd7cefa2b/config/application.rb#L189-L191

Not sure if still needed, but I guess probably yes?...